tests/snapshot/snapshots/snapshot__pretty_multiline_doc.ncl.snap

```diff
@@ -0,0 +1,21 @@
+---
+source: tests/snapshot/main.rs
+expression: snapshot
+---
+{
+  field | doc
+    m%%"
+    Contract to enforce the value is a string that represents a boolean literal. Additionally casts "True" to "true"
+    and "False" to "false". This shouldn't interpolate: %{null}
+    
+    For example:
+    ```nickel
+      ("True" | BoolLiteral) =>
+        "true"
+      ("hello" | BoolLiteral) =>
+        error
+      (true | BoolLiteral) =>
+        error
+    ```"%%
+     = 1,
+}
```

Comment:
Currently, we've rather written such things as:

```
{
  field | doc m%%"
    Contract to enforce the value is a string that represents a boolean literal. Additionally casts "True" to "true"
    and "False" to "false". This shouldn't interpolate: %{null}

    For example:
    ```nickel
      ("True" | BoolLiteral) =>
        "true"
      ("hello" | BoolLiteral) =>
        error
      (true | BoolLiteral) =>
        error
    ```
    "%%
     = 1,
}
```

That being said, nothing is set in stone, as we haven't decided on clear formatting rules yet.

Reply:
I've changed to your suggested formatting 👍